src/main/kotlin/no/nav/familie/ef/sak/vedtak/historikk/AndelHistorikkDto.kt

```diff
@@ -28,8 +30,10 @@ data class AndelHistorikkDto(
     val saksbehandler: String,
     val andel: AndelMedGrunnlagDto,
     val aktivitet: AktivitetType?,
+    val aktivitetBarnetilsyn: AktivitetstypeBarnetilsyn?, //TODO: Skal bli non-nullable
```

Comment:
Hvis vi splitter ut metadata for overgangsstønad og barnetilsyn til egne objekter kan de bli non-nullable, men sånn som de står nå så kan de ikke bli det :-)